index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website: Ademola Adebayo</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <section class="toolbar">
+    <div class="top-toolbar">
+      <p>10:30</p>
+      <div class="top-icons">
+        <i class="	fas fa-signal fa-0.5x"></i>
+        <i class="	fas fa-wifi fa-0.5x"></i>
+        <i class="	fa fa-battery-0 fa-0.5x"></i>       
+      </div>
+    </div>
+    <div class="btm-toolbar">
+      <h2>Smart City</h2>
+      <a href=""><i class=" fas fa-bars fa-2x"></i></a>
+    </div>
+  </section>
+  <section class="headline">
+    <div class="section-head">
+      <h1>Ademola Adebayo</h1>
+      <h3>Software Developer</h3> 
+    <div class="head-text">
+      <p>I can help you build a product,<br> feature or website Look through<br> some of my work and experience! If<br> you like what you see and have 
+        a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    <hr> 
+    </div>
+    <div class="lang-framework">
+      <div>
+        <p class="lang-col">Language</p>
+        <p>Javascript</p>
+        <p>HTML</p>
+        <p>Ruby</p>
+        <p>CSS</p>  
+      </div>
+      <div>
+        <p class="lang-col">Frameworks</p>
+        <p>Bootstrap</p>
+        <p>Ruby on Rails</p>
+        <p>React</p>
+        <p>Redux</p>
+        <p>State</p>
+      </div>
+    </div>
+    <div class="skills">
+      <p class="lang-col">Skills</p>
+        <p>Codekit</p>
+        <p>Github</p>
+        <p>Codepen</p>
+        <p>Terminal</p>
+        <p>Replit</p>  
+    </div>
+    <hr>
+    <div class="social">
+      <a href="#"><i class="fab fa-github fa-2x"></i></a>
+      <a href="#"><i class="fab fa-facebook fa-2x"></i></a>
+      <a href="#"><i class="fab fa-instagram fa-2x"></i></a>
+      <a href="#"><i class="fab fa-twitter fa-2x"></i></a>
+    </div>
+    <div class="dn-page">
+      <a href="#"><i class="fas fa-arrow-down fa-2x"></i></a>
+      <a href="#"><p>More</p></a>
+    </div>  
+  </section>
```

Comment:
- [ ] Kindly adjust the spacing between the elements and each other to match the Figma design.
- [ ] Kindly adjust the spacing around the header text (on the left and right) to match the Figma design.
- [ ] Please try to match the font style (color, family, weight) of the Figma design. (You can see what font they used and other important information that may help you achieve this on the right panel in the Figma design whenever you click on a part of the Figma design)
- [ ] Kindly add the background image for this section to match the Figma design. (You can click on the back ground images to export them and use them in your files).

| Figma design | Your design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/158820200-d4212610-878d-404f-90fd-5224f6b22009.png) | ![image](https://user-images.githubusercontent.com/62290545/158820298-e4b6426b-5e15-49da-9509-504e9c5027b7.png) |